code/modules/mob_spawn/ghost_roles/golem_roles.dm

```diff
@@ -52,6 +52,10 @@
 		if (policy)
 			to_chat(new_spawn, policy)
 		to_chat(new_spawn, "Build golem shells in the autolathe, and feed refined mineral sheets to the shells to bring them to life! You are generally a peaceful group unless provoked.")
+		var/static/list/allowed_areas
+		if(!allowed_areas)
+			allowed_areas = typecacheof(list(/area/icemoon, /area/lavaland, /area/ruin/powered/golem_ship))
```

Comment:
We can probably extend this to all of `/area/ruin`, yeah.

Reply:
```suggestion
			allowed_areas = typecacheof(list(/area/icemoon, /area/lavaland, /area/ruin))
```